apps/shopify/src/dataTransformer.js

```diff
@@ -107,25 +109,24 @@ export const productsToVariantsTransformer = (products) =>
 
 export const previewsToProductVariants =
   ({ apiEndpoint }) =>
-  ({ sku, id, image, product, title }) => {
-    const productIdDecoded = atob(product.id);
-    const productId =
-      productIdDecoded && productIdDecoded.slice(productIdDecoded.lastIndexOf('/') + 1);
-    return {
-      id,
-      image: get(image, ['src'], ''),
-      // TODO: Remove sku:id when @contentful/ecommerce-app-base supports internal IDs
-      // as an alternative piece of info to persist instead of the SKU.
-      // For now this is a temporary hack.
-      sku: id,
-      displaySKU: sku ? `SKU: ${sku}` : `Product ID: ${id}`,
-      productId: product.id,
-      name: title === DEFAULT_SHOPIFY_VARIANT_TITLE ? product.title : `${product.title} (${title})`,
-      ...(apiEndpoint &&
-        productId && {
-          externalLink: `https://${apiEndpoint}${
-            last(apiEndpoint) === '/' ? '' : '/'
-          }admin/products/${productId}`,
+    ({ sku, id, image, product, title }) => {
+      const productIdDecoded = atob(product.id);
+      const productId =
+        productIdDecoded && productIdDecoded.slice(productIdDecoded.lastIndexOf('/') + 1);
+      return {
+        id,
+        image: get(image, ['src'], ''),
+        // TODO: Remove sku:id when @contentful/ecommerce-app-base supports internal IDs
+        // as an alternative piece of info to persist instead of the SKU.
+        // For now this is a temporary hack.
+        sku: id,
+        displaySKU: sku ? `SKU: ${sku}` : `Product ID: ${id}`,
+        productId: product.id,
+        name: title === DEFAULT_SHOPIFY_VARIANT_TITLE ? product.title : `${product.title} (${title})`,
+        ...(apiEndpoint &&
+          productId && {
+          externalLink: `https://${apiEndpoint}${last(apiEndpoint) === '/' ? '' : '/'
+            }admin/products/${productId}`,
         }),
+      };
```

Comment:
If possible, I would isolate commits with formatting only: it makes scanning through the changes easier